File: legacy/src/unity/db/perl-lib/tables/review.pl
sub TblInit {

  my $tblDescription =
{
 name   => 'Review',
 table  => 'Reviews',
 authlevel => 'User',
 fields => { Review  => { longname=>'Text of Review', type=>'text' },
	     UserID  => { longname=>'User', type=>'search', hide=>1 },
	     AlbumID  => { longname=>'Album', type=>'search' },
	   },
 ID     => 1,
 sortby => 'Review',
 order  => [ qw( Review UserID AlbumID ) ],
 search	=> [ 'Review' ],
};
  return $tblDescription;
}

1;

File: app/models/review.js
var DB = require('bookshelf').DB,
		Album = require('./album'),
		User = require('./user');

var Review = DB.Model.extend({
	tableName: "Reviews",
	idAttribute: "ReviewID",

	album: function() {
		return this.belongsTo(Album.model, "AlbumID");
	},

	user: function() {
		return this.belongsTo(User.model, "UserID");
	}
});

var Reviews = DB.Collection.extend({
	model: Review,
});


exports.model = Review;
exports.collection = Reviews;
